backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -339,6 +339,9 @@ export class AdminResolver {
     let creations = await getUserCreation(user.id)
     if (contributionToUpdate.contributionDate.getMonth() === creationDateObj.getMonth()) {
       creations = updateCreations(creations, contributionToUpdate)
+    } else {
+      logger.error('Currently the month of the contribution cannot change.')
+      throw new Error('Currently the month of the contribution cannot change.')
```

Comment:
Currently this thrown error is not tested like in the `ContributionResolver.ts`.
If this should be then, please write a test.

If it is not neseccary in your eyes for the `AdminResolver.ts` then leave it like it is.

Reply:
This is only a temporal solution. Therefore I do not want to test it